test/Azure.Functions.Cli.Tests/E2E/StartTests.cs

```diff
@@ -1857,6 +1857,80 @@ await CliTester.Run(new RunConfiguration[]
             }
         }
 
+        [Theory]
+        [InlineData("dotnet-isolated", "debug")]
+        [InlineData("dotnet", "debug")]
+        public async Task Start_Host_SpecifieduserLogLevel_SuccessfulFunctionExecution(string WorkerRuntime, string UserLogLevel)
+        {
+            await CliTester.Run(new RunConfiguration[]
+            {
+                new RunConfiguration
+                {
+                    Commands = new[]
+                    {
+                        $"init . --worker-runtime {WorkerRuntime}",
+                        $"new --template Httptrigger --name HttpTrigger",
+                    },
+                    CommandTimeout = TimeSpan.FromSeconds(300),
+                },
+                new RunConfiguration
+                {
+                    Commands = new[]
+                    {
+                        $"start  --port {_funcHostPort} --userLogLevel {UserLogLevel}"
+                    },
+                    ExpectExit = false,
+                    Test = async (workingDir, p, _) =>
```

Comment:
I don't think this test is testing the log level - you should check to see if a debug log is present